snuba/subscriptions/entity_subscription.py

```diff
@@ -123,10 +124,16 @@ class TransactionsSubscription(BaseEventsSubscription):
     ...
 
 
+class MetricsCountersSubscription(SessionsSubscription):
+    MAX_ALLOWED_AGGREGATIONS: int = 3
+    disallowed_aggregations = ["having", "orderby"]
```

Comment:
Resurfacing my original question:

> At a high level I am not sure why you need to allow the group by aggregation. Could you please add more color to the PR description explaining which problem this PR is solving ?

Reply:
@fpacifici I updated the PR description already to explain why we need the group by column. Did you miss it or is it still not clear and I should I try to make it more expressive?